### ISSUE TITLE
[FR]: A column for subreddit name

### ISSUE THREAD

**Issue by Rex000** [martinrotter/rssguard#950]
### Brief description of the feature request

Suppose we have a reddit search feed: https://reddit.com/search.rss?q=guard

This returns posts from different subreddits. In the column options I ticked the "Feeds" checkbox. For individual subreddits it's fine. But for search links such as the one above, it doesn't work and only shows "reddit.com: search results - guard" in the "Feeds" column. This is useful, but is it possible to have a column type that displays only that particular subreddit's name that the article is from? Right now what I'm doing is ticking the "URL" checkbox as well which displays the entire url and then I have to look closely in it to find the subreddit name. So a separate column showing only the subreddit name would be very useful.

**Comment by martinrotter** [comment]
Well, this feed includes `<category term="bayarea" label="r/bayarea"/>` tags with names of individual subreddits.

At this point RSS Guard does not read these tags but support is planned in the future. Pull requests are welcomed. RSS Guard is project made in my free time with almost no income so you must understand I highly prioritize some tickets over others.

**Comment by Rex000** [comment]
Yes I do understand and I really appreciate the work you have done with this app.

**Comment by Rex000** [comment]
Yes I do understand and I really appreciate the work you have done with this app, and that you try to be as active as possible when it comes to fixing bugs or fulfilling requests. Thanks!

**Comment by Rex000** [comment]
Yes I do understand and I really appreciate the work you have done with this app, and that you try to be as active as possible whether it's about fixing bugs or fulfilling feature requests. Thanks!

**Comment by martinrotter** [comment]
I do not plan exactly implementing this by myself, but I plan to add something probably very similar in RSS Guard 5.0.0.
